supplemental-ui/.htaccess

```diff
@@ -1,5 +1,11 @@
 DirectoryIndex index.html index.asciidoc
 
+## set a policy to allow youtube.com video (avideo block macro reladed + wip)
+<IfModule mod_headers.c>
+Header set Content-Security-Policy "frame-src 'self' https://www.youtube.com "
+</IfModule>
```

Comment:
the CSP is usually one long String. For example my old blog uses this:
```html
<meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; img-src *; base-uri 'self'; connect-src 'self'; form-action 'self'; frame-ancestors 'none'">
```
So my question is what happens to the rest of the options if you set only `frame-src` but nothing else? This could cause other things to fail dependent on what the defaults are - I don't know unfortunately.  cc @vieiro

Reply:
I tried to look via curl locally and the best way to "edit" the existing CSP that come from ASF

is to append at begining of "frame-src" and "script-src" the url we want to allow what we wants

The CSP of main ASF could be found there.
https://github.com/apache/infrastructure-p6/blob/22223a2f370937bfbc7607bc965dea349f0befa6/data/roles/tlpserver.yaml#L568C6-L569C7
https://github.com/apache/infrastructure-p6/blob/22223a2f370937bfbc7607bc965dea349f0befa6/data/roles/tlpserver.yaml#L1455

by using edit like this we have a "valid" csp
Header edit Content-Security-Policy "frame-src" "frame-src https://www.youtube.com"
Header edit Content-Security-Policy "script-src"  "script-src https://www.youtube.com"

Reply:
I've tested live by commiting on asf-site branch the video will play.